### PR TITLE
Submit project proposals list for Summer 2021 of Open Source Promotion Plan

### PR DESCRIPTION
**Pull request by shenhonglei**
Signed-off-by: honglei <hongleishen@hotmail.com>

Submit project proposals list for Summer 2021 of Open Source Promotion Plan

Need to discuss and then improve this proposals list.

### CODE REVIEW

File: sig-advocacy-and-outreach/summer-ospp/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+| --- | --- | --- |
+| TODO | TODO | TODO |
+| TODO | TODO | TODO |
+| TODO | TODO | TODO |
+| TODO | TODO | TODO |
+
+## Students
+
+* Students application process
+* [Google Summer of Code portal](https://summer.iscas.ac.cn/)
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/README.md:15]: Google?

File: sig-advocacy-and-outreach/summer-ospp/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Mentors
+
+* [Summer 2021 of Open Source Promotion Plan Portal](https://summerofcode.withgoogle.com/)
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/README.md:19]: Same with here

File: sig-advocacy-and-outreach/summer-ospp/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+KubeSphere in [Summer 2021 of Open Source Promotion Plan](https://github.com/summer-ospp). You can find our proposals in this page. Or you also can submit your proposal to KubeSphere community.
+
+## Proposals
+
+| Project | Area | Skills to study/improve |
+| --- | --- | --- |
+| TODO | TODO | TODO |
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/README.md:7]: We can reuse the proposals from GSoC. Because we have not been accepted.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

File: sig-advocacy-and-outreach/summer-ospp/README.md (hunk 1 above)
Comment (shenhonglei) [reply sig-advocacy-and-outreach/summer-ospp/README.md]: typo

File: sig-advocacy-and-outreach/summer-ospp/README.md (hunk 3 above)
Comment (shenhonglei) [reply sig-advocacy-and-outreach/summer-ospp/README.md]: Has been reused

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
`.DS_Store` should be ignored. You can put it into the `.gitignore` file.

hi @kubesphere/sig-advocacy-and-outreach , Considering @shenhonglei is the owner of this program. I suggest that @shenhonglei be the owner of the directory `sig-advocacy-and-outreach/summer-ospp/`.

@shenhonglei , you can create an OWNER file which similar to [this](https://github.com/kubesphere/community/blob/master/sig-advocacy-and-outreach/OWNERS).

File: sig-advocacy-and-outreach/summer-ospp/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* [Students application process](students-application-process.md)
+* [Open Source Promotion Plan Portal](https://summer.iscas.ac.cn/))
+
+## Mentors
+
+* [Summer 2021 of Open Source Promotion Plan Portal](https://summer.iscas.ac.cn)
+
+## Contacts
+
+* TBD
```
Comment (FeynmanZhou) [inline sig-advocacy-and-outreach/summer-ospp/README.md:23]: What kind of Contact should we leave here? Email, mailing list, or WeChat group?

Comment (shenhonglei) [reply sig-advocacy-and-outreach/summer-ospp/README.md:23]: I suggest WeChat group.

**Comment by FeynmanZhou** [comment]
@shenhonglei Thanks for your contribution!

**Review by FeynmanZhou — APPROVED** [review]

**Review by LinuxSuRen — APPROVED** [review]

**Pull request merged**